### PR TITLE
Test for GCS on Local Backend

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: src/main/resources/standardTestCases/local_gcs/local_gcs.wdl [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+  output {
+    String result = read_string(stdout())
+  }
+}
+
+workflow local_gcs {
+  call makeGcsFile
+  call readGcsFileFromLocal { input: gcsFile = makeGcsFile.outputFile }
+}
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/local_gcs/local_gcs.wdl]: \n

File: src/main/resources/standardTestCases/local_gcs/local_gcs.wdl [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    backend: "Local"
+  }
+
+  output {
+    String result = read_string(stdout())
+  }
+}
+
+workflow local_gcs {
+  call makeGcsFile
```
Comment (mcovarr) [inline src/main/resources/standardTestCases/local_gcs/local_gcs.wdl]: can this be something like `makeGcsFileInJES` to be more explicit as to what's happening?

**Comment by mcovarr** [comment]
:+1: very cool  😄 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/89/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/local_gcs/local_gcs.wdl (hunk 2 above)
Comment (geoffjentry) [inline src/main/resources/standardTestCases/local_gcs/local_gcs.wdl]: I think you mean `makeGcsFileInJes`

**Comment by kshakir** [comment]
LGTM also 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/89/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**